vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.cc

```diff
@@ -1171,7 +1171,7 @@ bool CURRENT_RECONCILE::loadFromJson(const std::string & json) {
   if (!error) {
     error = !(d.HasMember("viewingId") && d["viewingId"].IsString() &&
       d.HasMember("fee") && d["fee"].IsDouble() &&
-      d.HasMember("category") && d["category"].IsInt());
+      d.HasMember("type") && d["type"].IsInt());
```

Comment:
we can't do this as old JSON file has category and now it will fail to load the file as you except for old files to have type. Please remove this check from here and do a migration path like here https://github.com/brave/brave-core/blob/4086c98b5fe95865efe85b7bc9a3a89b4f438132/vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.cc#L735

Reply:
Fixed (is fixed as part of base::JSONReader refactor but that did not help this PR)